services/TransactionService/src/Controller/TransactionController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Islandora\TransactionService\Controller;
+
+use Silex\Application;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class TransactionController {
+
+  public function create(Application $app, Request $request) {
+    return $app['api']->createTransaction();
```

Comment:
I like it simple, but i would provide the same fail over mechanic we have for the others routes. Same applies to the other routes. 
`try {
       $response = $app['api']->createTransaction();
     }
     catch (\Exception $e) {
       $app->abort(503, 'Chullo says "Fedora4 Repository Not available"');
     }
     return $response;
`